tensorflow_graphics/projects/mesh_rcnn/branches/mesh_head.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2020 The TensorFlow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Mesh refinement branch of Mesh R-CNN"""
+
+import tensorflow as tf
+from tensorflow.keras import layers as keras_layers
+
+import tensorflow_graphics.geometry.convolution.utils as conv_utils
+from tensorflow_graphics.nn.layer.graph_convolution import \
+  FeatureSteeredConvolutionKerasLayer
+from tensorflow_graphics.projects.mesh_rcnn.ops.vertex_alignment import \
+  vert_align
```

Comment:
Import statements need not follow character limit. You can leave them as long lines.

Reply:
+1 also fix in other PR